pkg/client/clientset_generated/internalclientset/typed/batch/internalversion/scheduledjob.go

```diff
@@ -50,7 +50,7 @@ type scheduledJobs struct {
 }
 
 // newScheduledJobs returns a ScheduledJobs
-func newScheduledJobs(c *BatchClient, namespace string) *scheduledJobs {
+func newScheduledJobs(c *BatchInternalversionClient, namespace string) *scheduledJobs {
```

Comment:
@liggitt I finally made a local commit that kept the internal one unqualified. It made the client-gen code difficult to read. Given that not many callers need to refer to the these types, and we are going to migrate to using versioned client, can you live with the long name?